app/Main.hs

```diff
@@ -60,7 +68,11 @@ mvActualToGolden testPath =
    in do
      actualFileExist <- doesFileExist actualFilePath
      when actualFileExist (do
-       putStrLn $ "  Replacing file: " ++ goldenFilePath ++ " with: " ++ actualFilePath
+       putStr "  Replacing file: "
+       putStrColor Yellow goldenFilePath
+       putStr " with: "
+       putStrColor Green actualFilePath
+       putStrLn ""
```

Comment:
If we take into account the changes mentioned above (`withColor` and `success` and `warning`), we could do:
```suggestion
       warning $ putStr goldenFilePath
       putStr " with: "
       success $ putStr actualFilePath
       putStrLn ""
```

Reply:
I like that idea @cptrodolfox Thanks a lot!